next.config.js

```diff
@@ -78,6 +78,15 @@ module.exports = withBundleAnalyzer({
             }),
           ],
         },
+        {
+          resourceQuery: /rss/,
+          use: [
+            ...mdx,
+            createLoader(function (src) {
+              return this.callback(null, src)
+            }),
+          ],
```

Comment:
This can be simply `use: mdx` since the loader is not doing anything 👍 

```suggestion
          use: mdx,
```

Reply:
Cool